docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,32 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-vm]]
```

Comment:
I don't agree because the ISO is just the packaging format for the host VM. If we put ourselves in the users' shoes, the ISO is the means to the VM which is the end.